packages/xo-server-sdn-controller/src/index.js

```diff
@@ -106,6 +106,22 @@ function updateNetworkOtherConfig(network) {
 // =============================================================================
 
 class SDNController extends EventEmitter {
+  /*
+  Attributes on created networks:
+  - `other_config`:
+    - `xo:sdn-controller:encapsulation`    : encapsulation protocol used for tunneling (either `gre` or `vxlan`)
+    - `xo:sdn-controller:pif-device`       : PIF on which the tunnels are created, must be physical and have an IP configuration
```

Comment:
What does it contain?

The UUID, the ref, the device of the PIF?

Reply:
The device.